compysition/actors/httpserver.py

```diff
@@ -92,6 +109,42 @@ def callback_wrapper(*args, **kwargs):
         return callback_wrapper
     '''
 
+class AcceptPlugin(object):
+    """**Bottle plugin that filters basic accept types that are processable by Compysition**"""
+
+    name = "atypes"
+    api = 3
```

Comment:
What's this `api = 3` used for, is that a Bottle plugin thing? This all looks good, just curious what this does.

Reply:
Yeah not having the api attribute throws a bottle error.  The name I just added to match the ContentTypePlugin but doesn't really affect anything that I've noticed.